src/app/App.js

```diff
@@ -36,6 +38,25 @@ class App extends Component {
     })
   }
 
+  checkForErrors = response => {
+    if (!response.ok) {
+      this.setState( {error: 'Oop, that/s no good 😅'} )
+    } else {
+      return response.json();
+    }
+  }
+
+  componentDidMount() {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
```

Comment:
One thing I was confused about - where's all that additional data supposed to come from? Budget, revenue, run time etc? After looking at the project spec - it looks like we'll need to make a second network request to access these additional movie details. The endpoint is the same, just need to add the movie's id number at the end. For example: `https://rancid-tomatillos.herokuapp.com/api/v2/movies/123456`